spec/lib/thredded_spec.rb

```diff
@@ -46,27 +46,27 @@
 end
 
 describe Thredded, '.user_path' do
-  after do
-    Thredded.user_path = nil
-  end
-
   it 'returns "/" if lambda is not set' do
-    expect(Thredded.user_path(nil)).to eq '/'
+    Thredded.user_path = nil
+    expect(Thredded.user_path(nil, nil)).to eq '/'
```

Comment:
What do you think about adding some assignments of/for the passed params to aid in clarity of what's being passed into the method?  something like

```ruby
Thredded.user_path(_view_context=nil, _user=nil)
```

As I'm reading through these tests it's a little harder to see what parameter is what. There are some where you can infer pretty easily but sometimes it's a little more work.

Thoughts?